src/components/number/Number.js

```diff
@@ -98,7 +98,7 @@ export default class NumberComponent extends Input {
   }
 
   get emptyValue() {
-    return '';
+    return '\0';
```

Comment:
I would like to understand why this change was made. This seems like it will cause regressions.